### PR TITLE
Always report  "InstantiateRuntime" event to telemetry

### PR DESCRIPTION
**Pull request by Zlatkovsky**
## Description

In PR #13305, @vladsud added telemetry to the runtime-instantiation event, but wrapped it in a `ReportIfTooLong` call.

I think it would be best to always report the instantiation duration. That way we can always rely on it for telemetry (today, we instead have some of our containers add the telemetry, but not all containers do -- so it would be great to be able to standardize our telemetry to always use the Fluid Framework's reported duration).

Here's an example of what the event looks like, compared with the existing event from one of the containers. The top one is the new one, the bottom one is the one that the container reported today (so by having the framework report it, we'd also eliminate the slight gap):
image

## Reviewer Guidance

@vladsud , note that for consistency with other event names, I also changed the capitalization of `instantiateRuntime` to `InstantiateRuntime`. To my knowledge, none of our main hosts have picked up the latest FF version yet, so it's unlikely we'll need to worry about the old name. But if you'd rather me keep the old capitalization, let me know.

### CODE REVIEW

**Review by vladsud — APPROVED** [review]

**Comment by Zlatkovsky** [comment]
@vladsud , @tylerbutler : since this change already has Vlad's approval, is there anything else it's waiting on before being merged? Ditto for related PR #13429

**Comment by vladsud** [comment]
@Zlatkovsky - I'd think you have merge permissions (if not, we should probably fix it). That said, I'm happy to merge it if that works for you :)

**Comment by Zlatkovsky** [comment]
@vladsud , it says that I'm not authorized to merge this pull request:
image

For now, if you can just merge the two PRs in, that'd be great. This one and #https://github.com/microsoft/FluidFramework/pull/13429

File: packages/loader/container-loader/src/containerContext.ts
Comment (anthony-murphy) [inline packages/loader/container-loader/src/containerContext.ts:327]: this change seems fine, but i wonder if more a sustainable long term approach would be to have enough observability points that a consumer could measure this directly if they care. nothing specifically to change here, just something to consider longer term.

Comment (Zlatkovsky) [reply packages/loader/container-loader/src/containerContext.ts:327]: @anthony-murphy , I agree in the general case. For this specific case, we're already doing a "ReportIfTooLong", so switching it reporting it all the time doesn't seem particularly problematic. And we're already intertwining FF telemetry with container telemetry anyway.

So yes, duly noted in general; but for this specific instance, going ahead and resolving this thread (and note there's a similar https://github.com/microsoft/FluidFramework/pull/13429 as well)

**Comment by vladsud** [comment]
@Zlatkovsky - I've merged both, and will take a look at permissions for future.

**Pull request merged**
